test/python/common.py

```diff
@@ -47,7 +47,10 @@ def import_pbtxt(self, pb_filename):
             tf.import_graph_def(graph_def)
         return graph
 
-    def with_ngraph(self, l, config=tf.ConfigProto()):
+    def with_ngraph(self, l, config=None):
+        # Passing config as None and then initializing it inside because mutable objects should not be used as defaults in python
```

Comment:
Comment too long. Plit this at 80 column..

Reply:
Split into 2 lines